dashboard/src/db.json

```diff
@@ -0,0 +1,1174 @@
+      "price": 4900,
+      "category": "ssd",
+      "brand": "gigabyte"
+    },
+    {
+      "id": "1hp5j5gm2geikr9kjkav",
+      "images": [
+        "https://content1.rozetka.com.ua/goods/images/big_tile/13467872.jpg",
+        "https://content2.rozetka.com.ua/goods/images/big_tile/13467913.jpg"
+      ],
+      "title": "Kingston SSD HyperX Fury 3D 240GB 2.5\" SATAIII TLC (KC-S44240-6F)",
+      "rating": 1.7,
+      "price": 1088,
+      "category": "ssd",
+      "brand": "kingston"
+    },
+    {
+      "id": "g1i5cvs5ivkr9kjkav",
+      "images": [
+        "https://content.rozetka.com.ua/goods/images/big_tile/16355467.jpg",
+        "https://content.rozetka.com.ua/goods/images/big_tile/16355360.jpg"
+      ],
+      "title": "Kingston SSD HyperX Fury 3D 480GB 2.5\" SATAIII 3D NAND TLC (KC-S44480-6F)",
+      "rating": 3.87,
+      "price": 1850,
+      "category": "ssd",
+      "brand": "kingston"
+    },
+    {
+      "id": "athlkfyqjqikr9kjkav",
+      "images": [
+        "https://content1.rozetka.com.ua/goods/images/big_tile/13467707.jpg",
+        "https://content2.rozetka.com.ua/goods/images/big_tile/13467728.jpg"
+      ],
+      "title": "Kingston SSD HyperX Fury 3D 120GB 2.5\" SATAIII TLC (KC-S44120-6F)",
+      "rating": 4.64,
+      "price": 869,
+      "category": "ssd",
+      "brand": "kingston"
+    },
+    {
+      "id": "9q71sd4okt9kr9kjkav",
+      "images": [
+        "https://content.rozetka.com.ua/goods/images/big_tile/169540018.jpg"
+      ],
+      "title": "Kingston NV1 500GB NVMe M.2 2280 PCIe 3.0 x4 (SNVS/500G)",
+      "rating": 2.96,
+      "price": 3860,
+      "category": "ssd",
+      "brand": "kingston"
+    },
+    {
+      "id": "pl8fxwgth9kr9kjkav",
+      "images": [
+        "https://content2.rozetka.com.ua/goods/images/big_tile/169540365.jpg"
+      ],
+      "title": "Kingston NV1 1TB NVMe M.2 2280 PCIe 3.0 x4 (SNVS/1000G)",
+      "rating": 5,
+      "price": 2600,
+      "category": "ssd",
+      "brand": "kingston"
+    },
+    {
+      "id": "4xij1knhfoukr9kjkav",
+      "images": [
+        "https://content2.rozetka.com.ua/goods/images/big_tile/59014647.jpg"
+      ],
+      "title": "Asus Xonar U7 MKII sound card",
+      "rating": 5,
+      "price": 2670,
+      "category": "sound_cards",
+      "brand": "asus"
+    },
+    {
+      "id": "54mt82049zlkr9kjkav",
+      "images": [
+        "https://content1.rozetka.com.ua/goods/images/big_tile/10687036.jpg",
+        "https://content2.rozetka.com.ua/goods/images/big_tile/10687086.jpg"
+      ],
+      "title": "Asus Strix Soar Sound Card (90YB00J0-M1UA00)",
+      "rating": 4.63,
+      "price": 2400,
+      "category": "sound_cards",
+      "brand": "asus"
+    },
+    {
+      "id": "js87e5b8rcqkr9kjkav",
+      "images": [
+        "https://content1.rozetka.com.ua/goods/images/big_tile/46331525.jpg"
+      ],
+      "title": "Asus Xonar U5 Sound Card",
+      "rating": 2.83,
+      "price": 1800,
+      "category": "sound_cards",
```

Comment:
**Issue**: The JSON data entries are added in a large block, which could lead to difficulties in version control systems and could affect performance when loading or parsing this file due to its size. <br> **Fix**: Split the JSON data into smaller chunks or separate files based on categories like 'video_cards', 'gaming_keyboards', etc. This modular approach aids in better organization and improves the manageability of data. <br> **Code Suggestion**: 
 ```
 + "category": "video_cards",
 + "category": "gaming_keyboards",
 + "category": "computer_mouse",
 + "category": "ssd",
 + "category": "sound_cards",
 - Store each category in separate JSON files or database tables.
 ```